staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/testing/v2/kms_plugin_mock.go

```diff
@@ -135,10 +133,8 @@ func (s *Base64Plugin) Start() error {
 // CleanUp stops gRPC server and the underlying listener.
```

Comment:
Can you fix all the `Start()` call sites that spin up new go routine and fail to check the error?